editor/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -114,7 +114,9 @@ class HierarchicalTermSelector extends Component {
 			.then( ( term ) => {
 				const hasTerm = !! find( this.state.availableTerms, ( availableTerm ) => availableTerm.id === term.id );
 				const newAvailableTerms = hasTerm ? this.state.availableTerms : [ term, ...this.state.availableTerms ];
-				const { onUpdateTerms, restBase, terms } = this.props;
+				const { onUpdateTerms, restBase, terms, slug } = this.props;
+				const termAddedMessage = slug === 'category' ? __( 'Category added' ) : __( 'Term added' );
```

Comment:
Too bad there's not a taxonomy label for this so we didn't need to hard-code it 😕 

https://developer.wordpress.org/reference/functions/get_taxonomy_labels/

Reply:
Maybe we can open a Trac ticket for this 🙂